meseq.py

```diff
@@ -153,11 +153,10 @@ def box(self, x, y, text):
         self.cr.stroke()
 
         # the text
-        self.text(x, y, text)
+        self.text(x, y, text, white_bg=False)
```

Comment:
ok for now.
I plan to modify it (in the future) with something like:
{ 'bgcolor' : '#fff', 'color' : '#000', ... }